### PR TITLE
#7 Team Base inconsistency fix

### PR DESCRIPTION
**Pull request by CoderSudaWuda**
Treats team base as a well. Doesn't allow a fast/high penetration entity to pass into the base.

### CODE REVIEW

File: src/config.ts [hunk 1]
```diff
@@ -19,7 +19,7 @@
 /** The build supported by the server. */
 export const buildHash: string = "6f59094d60f98fafc14371671d3ff31ef4d75d9e";
 /** The port the server is hosting its game server on. */
-export const serverPort = parseInt(process.env.PORT || "8080");
+export const serverPort = process.env.PORT || 8080; // << parseInt isn't needed
```
Comment (Nul-led) [inline src/config.ts:22]: This might not work in all cases because some platforms might insert the port as a string

File: src/index.ts [hunk 2]
```diff
@@ -117,7 +117,7 @@ server.listen(PORT, () => {
     // NOTES(0): As of now, both servers run on the same process (and thread) here
     // NOTES(1): This does not update the index.html - server list was always static, so you need to modify html first (see "Survival" in html)
     const jungle = new GameServer(wss, "jungle", "survival");
-    const sbx = new GameServer(wss, "ffa", "*");
+    const sbx = new GameServer(wss, "teams", "*");
```
Comment (Nul-led) [inline src/index.ts:120]: i don't get why u changed this lol

**Review by Nul-led — APPROVED** [review]
Lgtm :)

File: src/config.ts (hunk 1 above)
Comment (UE2020) [reply src/config.ts:22]: I thought all environment variables are strings?

Comment (Nul-led) [reply src/config.ts:22]: they are? Im not sure.. but u re prob correct lol

File: src/index.ts (hunk 2 above)
Comment (CoderSudaWuda) [reply src/index.ts:120]: i needed to test

File: src/config.ts (hunk 1 above)
Comment (CoderSudaWuda) [reply src/config.ts:22]: im pretty sure http already parseInt's  the ports

File: src/index.ts (hunk 2 above)
Comment (Nul-led) [reply src/index.ts:120]: ye but this doesnt seem necessary lol, whatever..

Comment (UE2020) [reply src/index.ts:120]: ABC can just change it back.

**Review by ABCxFF — CHANGES_REQUESTED** [review]
https://github.com/ABCxFF/diepcustom/blob/3e910976d0b91251f1cc4cf5d5c92d5fc3190807/src/Entity/Object.ts#L213 seems to be the only change that needs to be kept, if this is true, please delete the other changes.

**Comment by CoderSudaWuda** [comment]
will do 👍

**Pull request closed without merging**